episodes/severity-static.Rmd

```diff
@@ -0,0 +1,640 @@
+
+Let's use `{epiparameter}`:
+
+```{r, message=FALSE, warning=FALSE}
+# Get delay distribution
+onset_to_death_ebola <-
+  epiparameter::epidist_db(
+    disease = "Ebola",
+    epi_dist = "onset_to_death",
+    single_epidist = TRUE
+  )
+```
+
+To calculate the delay-adjusted CFR, we can use the `cfr_static()` function with the `data` and `delay_density` arguments.
+
+```{r}
+# Calculate the static CFR while adjusting for delays
+cfr_static(
+  data = ebola1976,
+  delay_density = function(x) density(onset_to_death_ebola, x)
+)
+```
+
+:::::::::::::::::: callout
+
+### Use the `<epidist>` class
+
+When using an `<epidist>` class object we can use this expression as a template `function(x) density(<EPIDIST_OBJECT>, x)`
+
+For distribution functions with parameters not available in `{epiparameter}`, we suggest you two alternatives: 
+
+- Create an `<epidist>` class object, to plug into other R packages of the outbreak analytics pipeline. Read the [reference documentation of `epiparameter::epidist()`](https://epiverse-trace.github.io/epiparameter/reference/epidist.html), or
+
+- Read `{cfr}` vignette for [a primer on working with delay distributions](https://epiverse-trace.github.io/cfr/articles/delay_distributions.html).
+
+::::::::::::::::::
+
+```{r,echo=FALSE}
+out_delay_adjusted <-
+  cfr_static(
+    data = ebola1976,
+    delay_density = function(x) density(onset_to_death_ebola, x)
+  )
+
+out_mean <- out_delay_adjusted %>% pull(severity_mean)
+out_low <- out_delay_adjusted %>% pull(severity_low)
+out_high <- out_delay_adjusted %>% pull(severity_high)
+```
+
+The delay-adjusted CFR indicated that the overall disease severity _at the end of the outbreak_ or with the _latest data available at the moment_ is `r out_mean` with a 95% confidence interval between `r out_low` and `r out_high`, slightly higher than the naive one.
+
+:::::::::::::::::::::::::::::::::::::::: challenge
+
+Load the `ebola_kikwit_1995` from `{outbreaks}` and estimate the delay-adjusted CFR using the appropriate distribution delay. Then:
+
+- Compare the naive and the delay-adjusted CFR solutions!
+
+:::::::::::::::::::: hint
+
+- Find the appropriate `<epidist>` object!
+
+::::::::::::::::::::
+
+:::::::::::::::::::: solution
+
+```{r}
+library(outbreaks)
+
+outbreaks::ebola_kikwit_1995 %>% glimpse()
+
+outbreaks::ebola_kikwit_1995 %>%
+  rename(
+    cases = onset,
+    deaths = death
+  ) %>%
+  cfr::cfr_static(
+    delay_density = function(x) density(onset_to_death_ebola, x)
+  )
+```
+
+::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::
+
+
+## An early-stage CFR estimate
+
+We just found that the difference between the _naive_ and _delay-adjusted_ CFR estimates using `cfr_static()` are _just slightly_ different. So, what is the main difference?!
```

Comment:
The first paragraph has a question about the difference between the naive and delay-corrected estimate that is not really answered later - what you are showing is the (low) impact on the estimated CFR, not the theory behind the difference. I may be nitpicking, but consider rephrasing? Also avoid using `?!`

Reply:
Should we locate the method callout here as a more appropriate reply? method callout added in https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/1ac361d2b3b7241cf43dc1c008225f534948b0a5

Reply:
The intention here is to emphasize the point of using {cfr} as a method to assess CFR at an early stage of an epidemic, and show this visually with the `cfr_rolling()`.

Since the method callout is placed before, I'll reiterate the method aspect briefly here.

Reply:
I added a sentence and a spoiler callout on the maximum likelihood step of the method, which aims to estimate the unbiased CFR by using the confirmed cases, confirmed deaths, and the underestimation factor (which is informed by the delay distribution presented in the first and previous method callout).

Added for review in https://github.com/epiverse-trace/tutorials-middle/pull/1/commits/e0662c2004fb98238b5131e380e8c68bc66fad3a

Reply:
Looks good